pkg/osquery/tables/osquery_exec_table/table.go

```diff
@@ -0,0 +1,83 @@
+// +build darwin
+
+// Package osquery_exec_table provides a table generator that will
+// call osquery in a user context.
+//
+// This is necessary because some macOS tables need to run in user
+// context. Running this in root context returns no
+// results. Furthermore, these cannot run in sudo. Sudo sets the
+// effective uid, but instead we need a bunch of keychain context.
+//
+// Resulting data is odd. If a user is logged in, even inactive,
+// correct data is returned. If a user has not ever configured these
+// settings, the default values are returned. If the user has
+// configured these settings, _and_ the user is not logged in, no data
+// is returned.
+
+package osquery_exec_table
```

Comment:
I think I want to leave this as is, since we might have a _non_ user exec table. But I think the function inside should be more clear. I'll update some naming around it

Reply:
Ended up taking that suggestion